docs/modules/ROOT/pages/clc-config.adoc

```diff
@@ -27,108 +29,117 @@ Usage:
+----
 
-|cluster.password
-|Any string
-|
-| Hazelcast Enterprise only
+== clc config import-cloud
 
-|cluster.discovery-token
-|Any string
-|
-|{hazelcast-cloud} only
+Imports configuration from a {hazelcast-cloud} CLI sample.
 
-|ssl.enabled
-|true / false
-|false
-|{hazelcast-cloud} or Hazelcast Enterprise
+In order to get the URL for the {hazelcast-cloud}, follow the steps below:
 
-|ssl.server
-|DOMAIN_NAME (must match SSL certificate)
-|
-|{hazelcast-cloud} (automatically set) or Hazelcast Enterprise
+1. On {hazelcast-cloud} console, visit:
++
+	Dashboard -> Connect Client -> CLI
 
-|ssl.skip-verify
-|true / false
-|false
-|{hazelcast-cloud} or Hazelcast Enterprise. Disables SSL host name and client-side verification. Never set to true in production.
+2. Copy the URL in box 2 and pass it as the second parameter.
+Make sure the text is quoted before running:
++
+[source,bash]
+----
+clc config import-cloud my-config "https://api..."
+----
 
-|ssl.ca-path
-|An absolute or relative path
-|
-|{hazelcast-cloud} or Hazelcast Enterprise. Sets the path to the CA certificate.
+.Global parameters
+[%collapsible]
+====
+include::partial$global-parameters.adoc[]
+====
 
-|ssl.key-path
-|An absolute or relative path
-|
-|{hazelcast-cloud} or Hazelcast Enterprise. Sets the path to the certificate key.
+== clc config import-java
 
-|ssl.key-password
-|Any string
-|
-|{hazelcast-cloud} or Hazelcast Enterprise. Sets the password for the certificate key.
+This command imports a subset of Hazelcast Java client configuration and saves it as CLC configuration.
 
-|log.path
-|An absolute or relative path or `stderr` to log to stderr.
-|`$CLC_HOME/logs/YYYY-MM-DD.log` where the date is today.
-|
+Java keystore and truststore referred in the XML configuration are converted to PEM files.
 
-|log.level
-|One of:
+Usage:
 
-. `error`
-. `warn`
-. `info`
-. `debug`
-| `info`
-|
+[source,bash]
+----
+clc config import-java [target] [source] [flags] [key=value...]
+----
 
-|===
+You can pass `key=value` pairs to override configuration items.
+
+.Configuration keys
+[%collapsible]
+====
+include::partial$configuration-items.adoc[]
+====
 
 Parameters:
 
-[cols="1m,1a,2a,1a"]
+[cols="1m,1m,2a,1m"]
 |===
 |Parameter|Required|Description|Default
 
 |`target`
 |Required
-|Name of the configuration or its path. If a name is used, the configuration is saved to `$CLC_HOME/configs/NAME`
-|N/A
+|Name or path to the configuration to be created
+|
+
+| `source`
+|Required
+|Path to the Java client XML configuration
+|
+
+|`--ca-alias`
+|Optional
+|Alias of the CA certificate in the Java truststore
+|
 
-|`key=value`
+|`--cert-alias`
 |Optional
-|Zero or more `config_item=value` pairs.
-|N/A
+|Alias of the certificate in the Java truststore
+|
+
+|`--key-alias`
+|Optional
+|Alias of the key in the Java keystore
+|
 
 |===
 
-Example output:
+## Limitations
+
+* Only PKCS12 keystores/truststores are supported, not JKS
```

Comment:
```suggestion
* JKS. Only PKCS12 keystores/truststores are supported
```

Reply:
I meant JKS type keystores are not supported.

Reply:
I've updated that section to:
```
=== Limitations

Only PKCS12 keystores/truststores are supported.

The following are not supported:

* JKS keystores/trustores
* <import> and <config-replacers> tags
* Variables in .xml configuration
```
Would that be OK?

Reply:
Even better than my suggestion! Thank you.

Reply:
Thanks!